src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -122,24 +126,23 @@ def app_create(cmd, client, resource_group, service, name,
         source=user_source_info)
 
     # create default deployment
-    logger.info("Creating default deployment with name '{}'".format(DEFAULT_DEPLOYMENT_NAME))
+    logger.info("Creating default deployment with name {}".format(DEFAULT_DEPLOYMENT_NAME))
     poller = client.deployments.create_or_update(
         resource_group, service, name, DEFAULT_DEPLOYMENT_NAME, properties)
 
     logger.info("Setting default deployment to production")
     properties = models.AppResourceProperties(
         active_deployment_name=DEFAULT_DEPLOYMENT_NAME, public=is_public)
-    poller.add_done_callback(lambda x: dump(x.resource()))
-    logger.info("Waiting for the default deployment completion")
+    logger.info("Waiting for the deployment {} create completion".format(DEFAULT_DEPLOYMENT_NAME))
     while poller.done() is False:

```

Comment:
As we discussed previously, the deployment poller should be wait after the the app.update?

Reply:
Make them parallel already.